function.rb

```diff
@@ -0,0 +1,9 @@
+
+def function(n)
```

Comment:
`function` doesn't seem like a meaningful name; it should probably be called 'factorial' or something